src/Blazor.Gitter.Client/Styles/_chatroom.scss

```diff
@@ -202,6 +205,26 @@
     }
 }
 
+.chat-room-messages--paused {
+    overflow-y: auto;
+}
+
+.chat-room-messages--paused::after {

```

Comment:
Paused overlay = feel free to modify if you like - just wanted to get something in there